script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_2.txt

```diff
@@ -70,14 +70,14 @@ I've heard some good news, and without that noisy Arihiko around, I think I can
 By the way.
 Why isn't she going back to her desk?
 <Page21>
-"Yumizuka-san, it's almost fifth period."
+"Yumizuka-san, it's almost time for fifth period." // suggestions: 'the fifth period', 'time for the fifth period', 'almost period five'
```

Comment:
People don't generally use `the` when referring to class periods, I feel. Current change is OK though.